spec.md

```diff
@@ -2644,7 +2801,10 @@ implies `hash(x) == hash(y)`.
 `int(x[, base])` interprets its argument as an integer.
 
```

Comment:
> `hash` fails if x, or any value upon which its hash depends, is unhashable.

If `hash` only works on strings, `x` can't depend on an unhashable value (or else it's already unhashable by virtue of its own type).

Reply:
Thanks, that was a relic of a more permissive hash function.